Paco-Server/src/com/google/sampling/experiential/server/CloudSqlSearchServlet.java

```diff
@@ -185,18 +201,25 @@ private String getStackTraceAsString(Throwable e) {
     return string;
   }
   
-  private boolean isTimezoneNeeded(String[] projCols){
-    boolean timezoneNeeded = false;
-    for (String eachCol : projCols) {
-      if (eachCol.equals(Constants.STAR)){
-        return false;
-      } else if(dateColumns.contains(eachCol)){
-        return true;
-      } 
+  private boolean isValidGroupBy(String[] selectColumnsArr, List<String> groupByCols) {
```

Comment:
It seems like this method belongs to a queryprocessor or something so that other clients can use it as well.